src/SentanceCaseCommand.ts

```diff
@@ -91,10 +91,10 @@ class SentanceCaseCommand extends UICommand {
     let isParagrphStart = false;
     if (prevCont && prevCont.trim().length > 0) {
```

Comment:
This if should be inverted. The cognitive complexity of this method is too high.